src/util/chunk_fs.js

```diff
@@ -19,14 +19,15 @@ class ChunkFS extends stream.Transform {
      *      target_file: object,
      *      fs_context: object,
      *      namespace_resource_id: string,
+     *      md5_enabled: boolean,
      *      stats: import('../sdk/endpoint_stats_collector').EndpointStatsCollector,
      * }} params
      */
-    constructor({ target_file, fs_context, namespace_resource_id, stats }) {
+    constructor({ target_file, fs_context, namespace_resource_id, md5_enabled, stats }) {
         super();
         this.q_buffers = [];
         this.q_size = 0;
-        this.MD5Async = config.NSFS_CALCULATE_MD5 ? new (nb_native().crypto.MD5Async)() : undefined;
+        this.MD5Async = md5_enabled ? new (nb_native().crypto.MD5Async)() : undefined;
```

Comment:
Can we have `this.MD5Async = (md5_enabled || config.NSFS_CALCULATE_MD5) ? new (nb_native().crypto.MD5Async)() : undefined;` so that we can preserve the older behaviour? Or preserving the older behaviour not a requirement?

Reply:
No, Actually our requirement is to remove `this.NSFS_CALCULATE_MD5` which is a global flag and replace it with our new flag `force_md5_enabled`

Reply:
Then I think we need to update the tests as well which might be setting this to true/false and then performing tests. Also if the intention is to get rid of `NSFS_CALCULATE_MD5` then maybe we can get rid of it from `config.js` as well?